policyengine_us/parameters/gov/states/vt/tax/income/credits/ctc/reduction/amount.yaml

```diff
@@ -0,0 +1,10 @@
+description: Vermont provides this maximum Child Tax Credit per qualifying child.
+values:
+  0000-01-01: 0
+  2023-01-01: 20
```

Comment:
change all value to 2022 - check 2021 tax form if the credit was mentioned - add 2022 and 2021 tax form references to all yaml files